Modules/socketmodule.c

```diff
@@ -7809,7 +7809,10 @@ PyInit__socket(void)
     PyModule_AddIntMacro(m, CAN_ISOTP_FORCE_RXSTMIN);
     PyModule_AddIntMacro(m, CAN_ISOTP_RX_EXT_ADDR);
     PyModule_AddIntMacro(m, CAN_ISOTP_WAIT_TX_DONE);
+#ifdef CAN_ISOTP_SF_BROADCAST
+    /* This constant is new and not always available */
     PyModule_AddIntMacro(m, CAN_ISOTP_SF_BROADCAST);
```

Comment:
I assume the version of the kernel could be checked on buildtime, couldn't it?

Reply:
Technically yes, but there is no guarantee that this symbol really is there when the kernel version is above a specific version.
If you install the headers yourself with `linux-headers-$(uname -r)`, the assumption works.

But if the installation of headers has been obscured, like with Raspberry Pi 's package `raspberrypi-kernel-headers`, this assumption no longer works. It just adds another assumption that the packages 
`raspberrypi-kernel-headers` and `raspberrypi-kernel` are in sync.

Reply:
Personally, I would argue that the mainline upstream kernel is the source of truth and this must be fixed downstream. No idea how the policy is on CPython.

Reply:
Well, lets wait for the Reviewer to comment on this. This should theoretically be  @tiran.

Reply:
aye.